lms/djangoapps/teams/templates/teams/teams.html

```diff
@@ -11,14 +11,18 @@
 
 <%include file="/courseware/course_navigation.html" args="active_page='teams'" />
 
-<div class="team-tab-content"></div>
+<div class="container">
+    <div class="course-wrapper">
+        <section class="course-content" id="course-content">
```

Comment:
Question: why did this change to `'#course-content'`? The teams tab exists outside of the LMS courseware.